docs/source/.lintr

```diff
@@ -5,5 +5,6 @@ linters: all_linters(
     # With "all_linters", these problems are also reported:
     todo_comment_linter = NULL, # TODO comments should be removed.
     undesirable_function_linter = NULL, # Function "library" is undesirable. As an alternative, use roxygen2's @importFrom statement in packages and `::` in scripts, instead of modifying the global search path.
+    one_call_pipe_linter = NULL
```

Comment:
```suggestion
    one_call_pipe_linter = NULL # Avoid pipe |> for expressions with only a single call.
```
Unless it would be less trouble to direct users to the other config for explanations on all of these.